site/operational-guide/concepts.md

```diff
@@ -0,0 +1,137 @@
+optic line severed. Weave is designed to allow peers and their
+containers to make maximum safe progress under conditions of
+partition, healing automatically once the partition is over.
+
+## IP Address Manager (IPAM)
+
+IPAM is the subsystem responsible for dividing up a large contiguous
+block of IP addresses (known as the IP allocation range) amongst peers
+so that individual addresses may be uniquely assigned to containers
+anywhere on the overlay network.
+
+When a new network is formed an initial division of the IP allocation
+range must be made. Two (mutually exclusive) mechanisms with different
+tradeoffs are provided to perform this task: seeding and consensus.
+
+### Seeding
+
+Seeding requires each peer to be told the list of peer names amongst
+which the address space is to be divided initially. There are some
+constraints and consequences:
+
+* Every peer added to the network _must_ receive the same seed list,
+  for all time, or they will not be able to join together to form a
+  single cohesive whole
+* Because the 'product UUID' and 'random value' methods of peer name
+  assignment are unpredictable, the end user must by necessity also
+  specify peer names
+* Even though every peer _must_ receive the same seed, that seed does
+  _not_ have to include every peer in the network, nor does it have to
+  be updated when new peers are added (in fact due to the first
+  constraint above it may not be)
+
+Example configurations are given in the section on deployment
+scenarios.
+
+### Consensus
+
+Alternatively, when a new network is formed for the first time peers
+can be configured to co-ordinate amongst themselves to automatically
+divide up the IP allocation range - this process is known as
+consensus, and is implemented via an election algorithm.  Consensus
+based bootstrapping requires each peer to be told the total number of
+expected peers (the 'initial peer count') in order to avoid small
+independent groups of peers from electing different leaders under
+conditions of partition.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+### Observers
+
+Finally, an option is provided to start a peer as an _observer_. Such
+peers do not require either a seed peer name list nor an initial peer
+count; instead they rely on the existence of other peers in the
+cluster which have been so configured. When an observer needs address
+space, it will ask for it from one of the peers which partook of the
+initial division, triggering consensus if necessary.
+
+Example configurations are given in the section on deployment
+scenarios.
+
+## Persistence
+
+Certain information is remembered between launches of weave (for
+example across reboots):
+
+* The division of the IP allocation range amongst peers
+* Allocation of addresses to containers
+
+The persistence of this information is managed transparently in a
+volume container but can be destroyed explicitly if necessary.
```

Comment:
I have added a 'Resetting persisted data' section to the tasks page and linked to it from here.